src/CoreWCF.Templates/tests/BasicTests.cs

```diff
@@ -66,10 +79,20 @@ public TestVariation Framework(string framework)
             return this;
         }
 
-        public static implicit operator object[](TestVariation testVariation) => new object[] { testVariation };
+        public void Deserialize(IXunitSerializationInfo info)
+        {
```

Comment:
Do you need to implement the deserialization code too? Based on my experience with some custom test runners in the WCF client codebase, this is needed when running the tests in VS as it does the test discovery (including enumerating the variations) in one process, serializes the test variations and passes it to another process to run them. Without the deserialization, the tests didn't run correctly or sometimes at all in the VS test runner. At least one issue I had to solve was isolated to debugging the tests.

Reply:
It was needed indeed for VS. This is fixed now. Thanks!